### DISCUSSION TITLE
[General] Sunrise alt DA

### DISCUSSION THREAD

**Discussion by kimurayu45z** [ethereum-optimism/developers#612] [General]
### Discussion Topic

Developer Support submission

github: https://github.com/SunriseLayer
stack: Cosmos SDK
project: Specialized DA Layer for Proof of Liquidity

### issue: 
Hello! We are Sunrise, a specialized DA layer for proof of liquidity.
We are developing customized DA adaptor for OP stack on Sunrise.
We would like to know if possible what would be the steps to achieve the following:

- Treat us to your customized DA ecosystem (https://docs.optimism.io/builders/chain-operators/features/alt-da-mode)
- Establishing a communication channel with you

### More about Sunrise
Sunrise is a specialized DA layer for Proof of Liquidity. Sunrise brings further innovation to the modular paradigm by introducing unique functionality: Fee Abstraction and Off chain blob data. Building with Sunrise enables the benefits of an aligned liquidity and security model by extending the Proof of Liquidity flywheel to the DA level.

Traction:
Sunrise Awakening Testnet has crossed 200,000 active users in its 6th WEEK.

Site:
https://sunriselayer.io/
X(Twitter):
https://x.com/SunriseLayer

Gitbook (White paper):
https://docs.sunriselayer.io/

Thanks in advance!
Yu Kimura - CEO of Sunrise

**Comment by imtipi** [comment]
> Establishing a communication channel with you

https://share.hsforms.com/1yENj8CV9TzGYBASD0JC8_gqoshb

you can fill the form to contact team
